src/gb/GB.cpp

```diff
@@ -3453,13 +3453,15 @@ bool gbReadGSASnapshot(const char* fileName)
     case 0x1b:
     case 0x1e:
     case 0xff:
-        read = fread(gbRam, 1, (gbRamSizeMask + 1), file);
-        toRead = (gbRamSizeMask + 1);
+        //read = fread(gbRam, 1, (gbRamSizeMask + 1), file);
```

Comment:
this should be safe to clean. no need for this to be checked (read, toRead)

Reply:
Yeah better to delete the old code entirely.